issue_maker.py

```diff
@@ -325,52 +337,71 @@ def delete_line():
     return f"Removed {cnt} instances of {song} by {artist} from unsupported.txt successfully."
 
 
+"""
+`github_webhook` function handles all notification from GitHub relating to the org. Documentation for the webhooks can
+be found at https://developer.github.com/webhooks/
+"""
+
+
+# helper function
+def validate_request(req):
```

Comment:
Maybe you should add this to utils.py then? That's the file for helper functions

Reply:
sure